content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -434,33 +435,33 @@ rules:
   verbs: ["get", "post"]
 ```
 
-### Mengacu Pada Subjek
+### Mengacu pada subjek
 
 RoleBinding atau ClusterRoleBinding mengikat sebuah Role ke subjek.
-Subjek dapat berupa kelompok, pengguna atau ServiceAccount.
+Subjek dapat berupa kelompok, pengguna atau {{< glossary_tooltip text="ServiceAccounts" term_id="service-account" >}}.
```

Comment:
nit: in the same context above, "kelompok" is written as "grup", pick one and make it consistent.

Reply:
Noted, I missed this one from the previous translation work..